gulpfile.babel.js

```diff
@@ -35,7 +35,7 @@ gulp.task('build', compile)
 gulp.task('watch', () => {
   compile()
   return watch([
-    path.normalize('components/**.js'),
+    path.normalize('components/*.js'),
```

Comment:
Or it's could be changed to `components/**/*.js` to watch for sub-folders, in case someone want reuse this boilerplate

Reply:
Can you do a PR on that ? :) thanks

Reply:
Sure